code/__DEFINES/mobs.dm

```diff
@@ -229,7 +229,7 @@
 //Sentience types, to prevent things like sentience potions from giving bosses sentience
 #define SENTIENCE_ORGANIC 1
 #define SENTIENCE_ARTIFICIAL 2
-// #define SENTIENCE_OTHER 3 unused
+#define SENTIENCE_HUMANOID 3
```

Comment:
Was this checked somewhere before hand but with `3` directly? Why is your code only updating flags?

Reply:
the code mostly checks if the potion's sentience type is the same as the mob's (i guess to make it so admins can edit the needed sentience?), and its not set anywhere to 3 from a basic check

Reply:
Ah, ok.